backend/README.md

```diff
@@ -1,7 +1,16 @@
-# Introduction 
+# Introduction
 This is the backend of Concordium Scan.
 
 # Getting Started
+
+This project depends on [concordium-net-sdk](https://github.com/Concordium/concordium-net-sdk) as a Git Submodule, which can be installed using:
+
+```
+git submodule update --init --recursive
+```
+
+Check out the readme of this dependency for how to build it.
```

Comment:
Is there a way to automate the build of the dependencies when this backend is built?

Reply:
Actually, it is already automated, the developer just needs to have rust and cargo installed, but don't want to maintain that information here as well as in the .NET SDK.